CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [2.12.3]
+### Changed
+- added `throttleKeypresses`, which in combination with `throttle` prevents to stop throttling by `keyup`
```

Comment:
I would have opted for something like:

### Added
- New `throttleKeypresses` option that can be used to prevent canceling of throttled events for individual key presses (i.e. on key up)